src/main/java/com/ontotext/ehri/deduplication/model/USHMMClassificationInstanceFactory.java

```diff
@@ -3,23 +3,32 @@
 import com.ontotext.ehri.deduplication.measures.JaroWinkler;
 import com.ontotext.ehri.deduplication.measures.Levenshtein;
 import com.ontotext.ehri.deduplication.measures.USHMMDate;
+import org.apache.commons.codec.EncoderException;
+import org.apache.commons.codec.language.DaitchMokotoffSoundex;
+import org.apache.commons.codec.language.bm.BeiderMorseEncoder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import types.Alphabet;
 import types.ClassificationInstance;
 import types.SparseVector;
 
-import java.util.ArrayList;
-import java.util.List;
+import java.util.*;
 import java.util.stream.Collectors;
 
-public class USHMMClassificationInstanceFactory {
```

Comment:
Add java doc explaining the purpose of the class.
Is the name really appropriate for this class? How about something which includes feature extraction?